src/plutus/cek.md

```diff
@@ -0,0 +1,136 @@
+# The CEK Machine
+
+This page presents the operational semantics of the Plutus language using an abstract machine - a [CEK machine](https://en.wikipedia.org/wiki/CEK_Machine).
+Although more complex than traditional reduction semantics, the CEK machine offers an efficient evaluation strategy that enables precise modeling of resource usage and cost.
+The production Haskell evaluator is based on the CEK machine, and it also provides a practical foundation for alternative implementations.
+
+The following listing defines some key concepts of the CEK machine.
+
+```text
```

Comment:
By the way, Github's Markdown supports embedding LaTeX:
https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions

Example:
```math
\begin{array}{lrclr}
    \textrm{State} & \Sigma & ::= & s;\rho \triangleright M \enspace | \enspace s \triangleleft V  \enspace |
       \enspace \diamond \enspace | \enspace \square\,V\\
    \textrm{Stack} & s      & ::= & f^*\\
\end{array}
```